src/PhpPact/Consumer/TestCase/MockServerTestCase.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace PhpPact\Consumer\TestCase;
+
+use PhpPact\Http\GuzzleClient;
+use PhpPact\Standalone\MockServer\MockServer;
+use PhpPact\Standalone\MockServer\MockServerConfig;
+use PhpPact\Standalone\MockServer\MockServerEnvConfig;
+use PhpPact\Standalone\MockServer\Service\MockServerHttpService;
+use PHPUnit\Framework\TestCase;
+
+class MockServerTestCase extends TestCase
+{
+    /** @var MockServerConfig */
+    protected $config;
+    /** @var MockServer */
+    private $mockServer;
+
+    protected function setUp()
+    {
+        $this->config = new MockServerEnvConfig();
+        $this->config
+            ->setPactFileWriteMode('merge')
+            ->setPactDir('D:/Development/PACT/pact-php');
+        $this->mockServer = new MockServer($this->config);
+        $this->mockServer->start();
```

Comment:
I thought the start and stop was in the listener (i.e. reason to move away from the bootstrap)

Reply:
You are right, I added this when playing around with the merge ability of the ruby mock server. I deleted the file. Thanks!